conf/20-look_and_feel

```diff
@@ -0,0 +1,3 @@
+# http://stackoverflow.com/questions/171563/whats-in-your-zshrc
```

Comment:
I will burn in FOSS-hell for this, but I don't think such a trivial line of configuration deserves attribution. If it makes you feel any better, it's in my zsh too.

Add a vim-modeline, © line and a short explanation of what that file does instead (see other files in conf/)